packages/java/tests/fix-lumo/src/main/templates/com/vaadin/fix/FixLumo.java

```diff
@@ -25,7 +25,6 @@
  * @deprecated internal API
  */
 @NpmPackage(value = "@vaadin/vaadin-themable-mixin", version = "${vaadin.components.npmVersion}")
-@NpmPackage(value = "@vaadin/polymer-legacy-adapter", version = "${vaadin.components.npmVersion}")
```

Comment:
The `"@vaadin/polymer-legacy-adapter": "24.8.0-alpha18"` entry should also be removed from all the `package.json` files, correct? Interesting that The [Gradle test failed](https://github.com/vaadin/hilla/actions/runs/15212450119/job/42789879088?pr=3515#step:10:3179) when it was looking for the `25.0.0.alpha1` verion of that package.